app/domain/errors.rb

```diff
@@ -484,6 +484,22 @@ module AuthnJwt
         code: "CONJ00102E"
       )
 
```

Comment:
@shulifink @hilagross Please review

Reply:
@nessiLahav it looks that this wasn't changed.
I added my comment to the ones who did